0.data_analysis_and_processing/mic_analysis/nbconverted/well_sirna_mic_analysis.py

```diff
@@ -240,13 +243,13 @@
 
     plt.gcf().set_size_inches(18, 10)
 
-    plt.xlabel("MIC", fontsize=13)
-    plt.ylabel("Density", fontsize=13)
+    plt.xlabel("MIC Score", fontsize=13)
+    plt.ylabel("Number of Wells", fontsize=13)
 
     plt.xticks(fontsize=12)
     plt.yticks(fontsize=12)
 
-    plt.title(f"Distributions of Maximal Information Coeficient (MIC)\nBetween DAPI and {stain} Features per Well", fontsize=16)
+    plt.title(f"Distributions of Maximal Information Coeficient (MIC) Scores\nBetween DAPI and {stain} Features per Well", fontsize=16)
 
     plt.xlim(0, 1)
     plt.ylim(0.0, 80.0)
```

Comment:
Consider updating the upper limit to 100.0 so that the bar in the GOLD plot isn't touching the top of the plot (gives it some room).

Reply:
I think it looks better with a little more room. I'll change that